src/apps/cart/index.md

```diff
@@ -8,104 +8,90 @@ swag:
-        <p>Get ready to take your online store to the next level with the reliable and feature-rich cart functionality of Shopware.</p>
+        <p align='justify'>A well-designed cart can significantly impact customer's shopping experience. Shopware's cart functionality handles all cart operations with ease and efficiency while offering customers a smooth and hassle-free shopping journey. You have the flexibility to customize pricing calculations, discounts, cart line items, and even implement custom cart operations to meet your unique business needs.</p>
         <h1>Capabilities</h1>
-        * [Calculate cart](/docs/guides/plugins/apps/app-scripts/cart-manipulation#calculating-the-cart)
-        * [Add line items](/docs/guides/plugins/apps/app-scripts/cart-manipulation#line-items)
-        * [Modify prices](/docs/guides/plugins/apps/app-scripts/cart-manipulation#price-definitions)
-        * [Add custom data](/docs/guides/plugins/apps/app-scripts/cart-manipulation#add-custom-data-to-line-items)
-        * [Add validations, errors and notifications](/docs/guides/plugins/apps/app-scripts/cart-manipulation#add-errors-and-notifications-to-the-cart)
-        * [Run rule-based cart scripts](/docs/guides/plugins/apps/app-scripts/cart-manipulation#rule-based-cart-scripts)
-        <h1>Change the look</h1>
-        <p>You can transform the appearance of your cart with our [template extensions](/docs/guides/plugins/plugins/storefront/customize-templates). By extending or overriding the appearance of the default storefront cart, you can create a unique and visually stunning shopping experience for your customers. Please note that only our default storefront merchants can take benefit from this tool to enhance the presentation of their online store.</p>
-        <p>Try our template extensions and elevate your cart's appearance to the next level.</p>
+    </template>
+    <template #im>
+        <img src="../../../public/landing/apps/cartstore.PNG"/>
     </template>
     <template #exposed>
         <SwagLandingCardList>
             <template #title>
-                Changing the cart behavior
-            </template>
-            <template #description>
-                Using <a href="/docs/guides/plugins/apps/app-scripts">App Scripts (cart scripts)</a>, you can <a href="/docs/guides/plugins/apps/app-scripts/cart-manipulation#overview">manipulate your cart</a> details to:
+                Get Ready To Elevate Your Online Store with Shopware's Feature-Rich Cart Functionalities
             </template>
             <template #cards>
                 <SwagLandingCard link="/docs/guides/plugins/apps/app-scripts/cart-manipulation#calculating-the-cart">
                     <template #title>Calculate cart</template>
+                    <template #sub><b>Auto-calculate - Total cost update with new products and discounts.</b></template>
```

Comment:
If `sub` needs to be bold, let's style it with CSS or add a class to the `SwagLandingCard` component.

Reply:
Removing bold! It was added added just for one page to check which was looking better. Text without bold itself is fine.